File: task1/Task1.js
// Вводим данные (для демонстрации захардкожено)
let A = [3, -5, 7, 0, 9, -1, 4];
let B = 4;

// Инициализация переменных
let sumPositive = 0, countPositive = 0, productGreaterB = 1, countGreaterB = 0;

// Обрабатываем массив
A.forEach(num => {
    if (num > 0) {
        sumPositive += num;
        countPositive++;
    }
    if (num > B) {
        productGreaterB *= num;
        countGreaterB++;
    }
});

// Результаты
console.log("Сумма положительных элементов:", sumPositive);
console.log("Количество положительных элементов:", countPositive);
if (countGreaterB > 0) {
    console.log("Количество элементов больше B:", countGreaterB);
    console.log("Произведение элементов больше B:", productGreaterB);
} else {
    console.log("Нет элементов больше заданного B");
}

File: task1/Task1.py
# Вводим данные
N = int(input("Введите размер массива: "))
A = [int(input(f"Элемент {i+1}: ")) for i in range(N)]
B = int(input("Введите число B: "))

# Инициализация переменных
sum_positive = 0
count_positive = 0
product_greater_b = 1
count_greater_b = 0

# Обрабатываем массив
for num in A:
    if num > 0:
        sum_positive += num
        count_positive += 1
    if num > B:
        product_greater_b *= num
        count_greater_b += 1

# Результаты
print("Сумма положительных элементов:", sum_positive)
print("Количество положительных элементов:", count_positive)
if count_greater_b > 0:
    print("Количество элементов больше B:", count_greater_b)
    print("Произведение элементов больше B:", product_greater_b)
else:
    print("Нет элементов больше заданного B")